Update delegate-mtls-support.md (#1563)
harness/developer-hub: docs/platform/2_Delegates/secure-delegates/delegate-mtls-support.md

- STRICT - only mTLS delegates are accepted. Any non-mTLS delegates are blocked. 

:::note

> mTLS is an advanced feature. Contact Harness Support to enable it and only supported on immutable delegate image.

Reviewer: ```suggestion
mTLS is an advanced feature and is only supported on immutable delegate images. Contact Harness Support to enable it.
```

### Create a CA certificate

> - Use the following OpenSSL comment to create a test CA certificate with no password and 25+ years of validity. The public part of the CA certificate (ca.crt) should be provided to Harness to enable mTLS.

Reviewer: ```suggestion
- Use the following OpenSSL comment to create a test CA certificate with no password and 25+ years of validity. You must provide the public portion of the CA certificate (ca.crt) to Harness to enable mTLS.
```

3. Using the previously created CA certificate with the certificate signing request, create the final signed client certificate:

    `openssl x509 -req -sha256 -days 9999 -extfile client.cnf -extensions v3_req \ -CAcreateserial -CA "ca.crt" -CAkey "ca.key" \ -in "client.csr" -out "client.crt"`
    
:::note

> The client.crt & client.key will be provided to the delegate YAML when installing the delegate.

Reviewer: ```suggestion
You provide the client.crt and client.key to the delegate YAML when you install the delegate.
```

:::

> 4. Once the certificates are created, provide the public cert of the CA certificate to Harness support. 

Reviewer: ```suggestion
4. After you create the certificates, provide the public cert of the CA certificate to Harness support. 
```

> 5. Furthermore, provide a unique prefix where the mTLS endpoint to be hosted at (this can be anything, but has to be unique)

Reviewer: ```suggestion
5. Provide a unique API prefix to the location where your mTLS endpoint is hosted. You can set any prefix for your directory path, but it must be unique.
```
